packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -505,6 +520,35 @@ export default {
             }
         },
 
+        /**
+         * Look up the geo details for the customers address, skip on failure.
+         *
+         */
+        async lookupGeoLocation () {
+            const postData = { addressLines: [] };
+            Object.keys(this.address).forEach(propName => {
+                if (this.address[propName]) {
+                    postData.addressLines.push(this.address[propName]);
+                }
+            });
+
+            try {
+                if (postData.addressLines.length) {
```

Comment:
Are we sure we need this validation here? (asking, not suggesting 😄)

Reply:
I didn't want to make an api if for any reason the address fields are empty plus the api fails if <1 address line.

Reply:
Do we want to log anything if this `if` statement fails?

Reply:
Not currently